test/e2e/checkpoint_test.go

```diff
@@ -1355,7 +1355,7 @@ var _ = Describe("Podman checkpoint", func() {
 		result := podmanTest.Podman([]string{"container", "checkpoint", "test_name"})
 		result.WaitWithDefaultTimeout()
 		Expect(result).Should(Exit(125))
-		Expect(result.ErrorToString()).To(ContainSubstring("criu failed"))
+		Expect(result.ErrorToString()).To(ContainSubstring("failed: exit status 1"))
```

Comment:
Could this be one of those stdout-vs-stderr things? Because `criu failed` is definitely part of the diagnostic. Is there a Golang way to check both streams? I think "criu failed" is a more representative and safe check than "failed: exit status 1". But I don't have a strong opinion about it, it's more important to get CI working again.

(PS thank you for getting to this so quickly)

Reply:
> Could this be one of those stdout-vs-stderr things?

I think only the last message (which contains "failed: exit status 1") is sent to the remote client. For example:
```
# podman-remote container checkpoint some-mysql
Error: `/usr/bin/crun checkpoint --image-path /var/lib/containers/storage/overlay-containers/cd337d16f713a4d430c28511fef9934cf72aed246540fed7def7b20f086d786d/userdata/checkpoint --work-path /var/lib/containers/storage/overlay-containers/cd337d16f713a4d430c28511fef9934cf72aed246540fed7def7b20f086d786d/userdata cd337d16f713a4d430c28511fef9934cf72aed246540fed7def7b20f086d786d` failed: exit status 
```

The `criu failed` message is logged but not sent to the client.